.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "editor.formatOnSave": true,
+  "editor.defaultFormatter": "Dart-Code.dart-code"
+}
```

Comment:
Nice! I personally have these in my user settings so they apply to any `dart/flutter` project I work on, but sharing on this project seems great as well. Thanks for the update.

I also have them scoped to `["dart"]` to avoid having a formatter try to run on other non-dart, files.

```json
"[dart]": {
    "editor.formatOnSave": true,
    "editor.formatOnType": true,
    "editor.rulers": [
      80
    ],
    "editor.selectionHighlight": false,
    "editor.suggest.snippetsPreventQuickSuggestions": false,
    "editor.suggestSelection": "first",
    "editor.tabCompletion": "onlySnippets",
    "editor.wordBasedSuggestions": "off",
    "editor.defaultFormatter": "Dart-Code.dart-code"
  },
```

Reply:
I updated with language scopes. I think your personal rules should override everything not specified so I provided a minimal set here but let me know if they dont